billing_agreements.go

```diff
@@ -9,18 +9,16 @@ import (
 // Endpoint: POST /v1/billing-agreements/agreement-tokens
 func (c *Client) CreateBillingAgreementToken(
```

Comment:
This is no more backwards compatible. Unless this API does not work, and is broken, we should maybe make a new method

Reply:
Added new function for backwards compatibility

Reply:
@cameronjarnot1 qq, does the current method work and is deprecated or does it just not work >

Reply:
works and is deprecated

Reply:
also still returns the same thing whether billing-agreements/agreement-tokens or payments/billing-agreements